tester_app/client/server.js

```diff
@@ -0,0 +1,54 @@
+/* eslint-disable no-console, func-names, no-var */
+var bodyParser = require('body-parser');
+var webpack = require('webpack');
+var WebpackDevServer = require('webpack-dev-server');
+var jade = require('jade');
+var sleep = require('sleep');
+var config = require('./webpack.client.hot.config');
+
+var name = "CHANGE ME!";
+
+var server = new WebpackDevServer(webpack(config), {
+  publicPath: config.output.publicPath,
+  hot: true,
+  historyApiFallback: true,
+  stats: {
+    colors: true,
+    hash: false,
+    version: false,
+    chunks: false,
+    children: false,
+  },
+});
+
+server.app.use(bodyParser.json(null));
+server.app.use(bodyParser.urlencoded({extended: true}));
+
+server.app.get('/hello_world.json', function(req, res) {
```

Comment:
This all needs to be commented out, as we're not doing any async for v1.

this will come later when we add the 

--async option

so let's comment out the get and post, but put in comments to see tutorial for an example of how to do this.